CleverTap/Plugins/iOS/CleverTapTemplatePresenter.mm

```diff
@@ -1,20 +1,16 @@
-//
-//  CleverTapTemplatePresenter.m
-//
-//  Created by Nikola Zagorchev on 22.10.24.
-//
-
 #import "CleverTapTemplatePresenter.h"
 #import "CleverTapUnityManager.h"
+#import "CleverTapMessageSender.h"
+#import "CleverTapUnityCallbackInfo.h"
 
 @implementation CleverTapTemplatePresenter
 
-- (void)onPresent:(nonnull CTTemplateContext *)context { 
-    UnitySendMessage([kCleverTapGameObjectName UTF8String], [kCleverTapCustomTemplatePresent UTF8String], [context.templateName UTF8String]);
+- (void)onPresent:(nonnull CTTemplateContext *)context {
+    [[CleverTapMessageSender sharedInstance] send:CleverTapUnityCallbackCustomTemplateClose withMessage:context.templateName];
```

Comment:
The event should probably be something like CleverTapUnityCallbackCustomTemplatePresent

Reply:
Fixed